.gitignore

```diff
@@ -52,3 +52,25 @@ docs/_build/
 
 # PyBuilder
 target/
+imager/media/comet_on_3_august_2014.png
+imager/media/gww5Fq2.jpg
+imager/media/iapetus3_cassini_big.jpg
+imager/media/1025382_687827634576731_251225928_o_0hBnaKH.jpg
+imager/media/1025382_687827634576731_251225928_o_lMp3ZkX.jpg
+imager/media/1f4d65843024054cd2702475f95b72ff6dd1b9da_mjgTAwy.jpeg
+imager/media/2013-07-27_03_37_41_SgN4toh.gif
+imager/media/gegeg_OI5fWql.jpg
+imager/media/IMG_4911_k58zn2v.PNG
+imager/media/cache/07/e2/07e2850a7c4614dd4777446bb41c5a56.jpg
+imager/media/cache/22/8f/228f73ba310e89b5b33f41ea702f9c2a.jpg
+imager/media/cache/7e/0a/7e0acda0bc1734027bb1d2b0ab4be42c.jpg
+imager/media/cache/95/3a/953ac190f1ae308ffd3c4cdb427ec393.jpg
+imager/media/cache/9e/10/9e100b46463d97ae60ec8af95b9c5209.jpg
+imager/media/butterfly_v4fEuab_q5Y3WKL.jpg
+imager/media/M31_HALRGBpugh.jpg
+imager/media/cache/30/20/3020d7ceb8589e26a62fd78164d58e08.jpg
+imager/media/cache/ea/b5/eab5e8c54b3f051d70657d22bee9ed1c.jpg
+imager/media/JFhXH.png
+imager/media/june22moonhalo_argerich.jpg
+imager/media/cache/2e/4f/2e4f76350c12c808c512add4a7fffc41.jpg
+1dc3b83bbf003b13db718cda09692248.jpg
```

Comment:
this is a shitty way to omit stuff in media.  Just ignore the whole directory and everything in it.